### PR TITLE
Shortened DESCRIPTION variables

### PR DESCRIPTION
**Pull request by jonasstein**
Fix repoman issues about too long DESCRIPTION variables

### CODE REVIEW

File: dev-libs/libtommath/libtommath-0.42.0-r1.ebuild [hunk 1]
```diff
@@ -5,7 +5,7 @@ EAPI=4
 
 inherit autotools eutils multilib toolchain-funcs
 
-DESCRIPTION="highly optimized and portable routines for integer based number theoretic applications"
+DESCRIPTION="optimized and portable routines for integer theoretic applications"
```
Comment (nicolasbock) [inline dev-libs/libtommath/libtommath-0.42.0-r1.ebuild]: To be consistent, I would capitalize the `o`.

File: dev-libs/libtommath/libtommath-1.0-r1.ebuild [hunk 2]
```diff
@@ -5,7 +5,7 @@ EAPI=5
 
 inherit autotools eutils multilib toolchain-funcs
 
-DESCRIPTION="highly optimized and portable routines for integer theoretic applications"
+DESCRIPTION="optimized and portable routines for integer theoretic applications"
```
Comment (nicolasbock) [inline dev-libs/libtommath/libtommath-1.0-r1.ebuild]: Capital `o`.

File: dev-libs/libtommath/libtommath-1.0.ebuild [hunk 3]
```diff
@@ -5,7 +5,7 @@ EAPI=5
 
 inherit autotools eutils multilib toolchain-funcs
 
-DESCRIPTION="highly optimized and portable routines for integer theoretic applications"
+DESCRIPTION="optimized and portable routines for integer theoretic applications"
```
Comment (nicolasbock) [inline dev-libs/libtommath/libtommath-1.0.ebuild]: Capital `o`.

Comment (jonasstein) [inline dev-libs/libtommath/libtommath-1.0.ebuild]: Thank you for your comment. I fully agree and changed it.

**Comment by nicolasbock** [comment]
Thanks!

**Pull request closed without merging**